src/Microsoft.Extensions.FileSystemGlobbing/Abstractions/DirectoryInfoBase.cs

```diff
@@ -2,16 +2,32 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.Collections.Generic;
-using System.IO;
 
 namespace Microsoft.Extensions.FileSystemGlobbing.Abstractions
 {
+    /// <summary>
+    ///     Represents a directory
```

Comment:
What's with the extra leading space? We don't do that elsewhere in this project: https://github.com/aspnet/FileSystem/blob/dev/src/Microsoft.Extensions.FileProviders.Abstractions/IFileInfo.cs#L10